lua/entities/magicbutton.lua

```diff
@@ -305,57 +325,321 @@ local function MagicOutcomeBountyAndPrize(ply)
+
+        if (math.random(1, 20) == 1) then
+            ply = table.Random(player.GetAll())
+            someoneelse = true
+        end
+
+        KLEINER_BULLIES[ply:SteamID()] = 5000
+        if (someoneelse) then return "and it sent the kleiner mob after " .. ply:Nick() .. "! ;antikleiner;" end
+
+        return "and it made them accidentally say a anti-kleiner slur! Now you're gonna get it! ;antikleiner;"
+    end
+end
+
+local function MagicOutcomeKleinerTeleported(ply)
+    if (KLEINER_NPCS and table.Count(KLEINER_NPCS) > 0) then
+        local someoneelse
+
+        if (math.random(1, 20) == 1) then
+            ply = table.Random(player.GetAll())
+            someoneelse = true
+        end
+
+        for k, v in pairs(ents.FindByClass("kleiner")) do
+            v:TeleportSafe(ply:GetPos())
+        end
+
+        if (someoneelse) then return "and it teleported all kleiners to the player [white]" .. ply:Nick() .. ";hahaha;" end
+
+        return "and it teleported all kleiners to them! ;kleinerfortnite;"
+    end
+end
+
+local landmarks = {
+    ["Near The Drunken Clam"] = Vector(-2518, -802, 64),
+    ["Near Trump Tower"] = Vector(-2528, -66, 64),
+    ["Near Sushi Theater"] = Vector(-2537, -1613, 104),
+    ["Near The Pit"] = Vector(-126, -1355, 19),
+    ["Somewhere on the roof"] = Vector(-1179, 1020, 522),
+    ["Somewhere on the roof"] = Vector(-234, 990, 633),
+    ["Somewhere on the roof"] = Vector(1616, 810, 606),
+    ["Somewhere on the roof"] = Vector(-427, 1352, 688),
+    ["Near Mini Golf"] = Vector(2113, -589, 119),
+    ["Near AFK Corral"] = Vector(2831, 1062, 129),
+    ["Somewhere behind the theater"] = Vector(1780, 1730, 108),
+    ["Near the cabin"] = Vector(-107, 2930, 97),
+    ["Near SportZone"] = Vector(1667, -1918, 85),
+}
+
+local function MagicOutcomeButtonSpawn(ply)
+    local button = ents.Create("magicbutton")
+    button:Spawn()
+    local trace = button:FindHidingSpot()
+
+    if (trace) then
+        button:MoveToTraceResult(trace)
+    end
+
+    if (IsValid(button)) then
+        local loc = Location.Find(button)
+        local locd = Location.GetLocationByIndex(loc or -1)
+        local locname = locd.Name
+
+        if (locname == "Outside") then
+            local nearest = 1000000
+            local nearestname
+            local secondnearestname
+
+            for k, v in pairs(landmarks) do
+                if (v:Distance(button:GetPos()) < nearest) then
+                    secondnearestname = nearestname
+                    nearestname = k
+                    nearest = v:Distance(button:GetPos())
+                end
+            end
+
+            locname = locname .. " (" .. nearestname .. ")"
+        end
+
+        return locname or "Somewhere stupid"
+    end
+end
+
+local function MagicOutcomeSpawnObject(ply, button)
+    local classes = {}
 
-    timer.Create("KLEINER_GOD_EXPIRE", 60 * 15, 1, function()
-        KLEINER_OVERRIDE_TARGET = nil
+    classes["sent_ball"] = {
```

Comment:
is this in the base gamemode or only in sandbox?

Reply:
i think it's installed to the game's base lua/entities dir